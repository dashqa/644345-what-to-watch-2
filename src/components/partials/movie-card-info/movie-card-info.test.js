import React from 'react';
import renderer from 'react-test-renderer';
import MovieCardInfo from './movie-card-info';

const props = {
  name: `A Star Is Born`,
  genre: `Drama`,
  released: 2010,
};

it(`Movie card info correctly renders after relaunch`, () => {
  const tree = renderer.create(<MovieCardInfo {...props}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
