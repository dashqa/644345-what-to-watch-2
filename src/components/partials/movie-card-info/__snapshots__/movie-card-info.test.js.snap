// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie card info correctly renders after relaunch 1`] = `
Array [
  <h2
    className="movie-card__title"
  >
    A Star Is Born
  </h2>,
  <p
    className="movie-card__meta"
  >
    <span
      className="movie-card__genre"
    >
      Drama
    </span>
    <span
      className="movie-card__year"
    >
      2010
    </span>
  </p>,
]
`;
